tools/mapping/gen_mapping_files/runoff_to_ocn/README

```diff
@@ -102,16 +85,78 @@ Settings:
   step2                 = computes smooth map (default: .true.)
   step3                 = multiply two maps together (default: .true.)
 
-Output fields:
+4. Output fields
   file_nn     = nearest neighbor mapping file (default: 'nn.nc')
   file_smooth = smoother mapping file (default: 'smooth.nc')
   file_new    = combined file (default: 'nnsm.nc')
 
+========================
+WHAT THESE TOOLS PROVIDE
+========================
 
-==========
-HOW TO RUN
-==========
+The runoff_map executable generates three maps:
 
-Execute the binary ./runoff_map (again, this will read the runoff_map.nml
-namelist file - see above section for tips on creating the file)
+1) A nearest neighbor mapping from the rof grid to the ocean grid.
+   * For river runoff, it may be desireable to limit the destination grid
+     so that runoff only enters at coastal points. The optional file_ocn_coast_mask
+     variable in the namelist allows you to specify two ocean grid files; one for
+     the full ocean and one for just the coast
+     variable can be used to point to an ocean grid 
+   * The optional file_ocn_coast_mask file allows mapping to a subset of ocean
```

Comment:
This bullet point seems to repeat some of the earlier one; can they be combined?